docs/en/remote-flow.rst

```diff
@@ -232,16 +238,27 @@ Below a non-normative example of the HTTP Request to the status endpoint, where
 
 When the status endpoint returns **200 OK**, it means that the User authentication is successful and the JavaScript code will use the returned location where the user-agent will be redirect to continue the navigation.
 
+Even if an adversary were to steal the random value used in the request to the status endpoint, their user-agent would be rejected due to the missing cookie in the request.
+
+
 Request Object Details
 ----------------------
 
 Below a non-normative example of HTTP request made by the Wallet Instance to the Relying Party.
```

Comment:
Normative details of the Request URI request are missing

Reply:
see my previous comment. I'd prefer to not redefine those parameters but reference the section where they are defined.